app/templates/hobbies-Josh.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    
```

Comment:
<body>
<div class= "Josh's Hobbies">
    <H1>Josh's Hobbies</H1>
    <H2>Video Games</H2>
    <p> One of the things Josh likes to do is play video games like Lost Ark.</p>
    <img src="https://cdn.discordapp.com/attachments/779215689108946948/982469968249638962/Screenshot_1025.png" alt="">
    <H2>Sneakers</H2>
    <p>He's also into sneakers. Below is one of his favorite pairs</p>
    <img src="https://cdn.discordapp.com/attachments/779215689108946948/982471804486574090/IMG_20210524_120458.jpg" alt="">
</div>
</body>